src/transformers/visitors/toCircuitVisitor.ts

```diff
@@ -1193,9 +1226,34 @@ const visitor = {
 
   IfStatement: {
     enter(path: NodePath, state: any) {
-      const { node, parent } = path;
+      const { node, parent, scope } = path;
       let isIfStatementSecret;
```

Comment:
this variables has type boolean, you can mark this as boolean as `let isIfStatementSecret : boolean`

Reply:
I've added this in.